cmake/catkin_download.cmake

```diff
@@ -0,0 +1,146 @@
+#
+# Download a file containing data from a URL.
+#
+# It is commonly used to download larger data files which should not be
+# stored in the repository.
+#
+# .. note:: It is not recommended to rely on downloaded data during
+#   a configure / make cycle since this prevents building the package
+#   when no network connectivity is available.
+#
+# .. note:: The target will be registered as a dependency
+#   of the "download_extra_data" target.
+#
+# :param target: the target name
+# :type target: string
+# :param url: the url to download
+# :type url: string
+#
+# :param DESTINATION: the directory where the file is downloaded to
+#   (default: ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_SHARE_DESTINATION})
```

Comment:
The default value in the code seems to be different: https://github.com/ros/catkin/pull/716/files#diff-f6d46f8e02eefd6f88ad660967bc8182R44